publish/README.md

```diff
@@ -19,3 +19,17 @@ As with all npm packages, there's a freely available CDN via unpkg.com:
     Rive({
          locateFile: (file) => 'https://unpkg.com/rive-canvas@0.6.7/'+file,
     }).then(...)
+    
+## In a Typescript Project
+
+```typescript
+import Rive, { File } from 'rive-canvas';
+
+async function loadRivFile(filePath: string): Promise<File> {
+  const req = new Request(filePath);
+  const loadRive = Rive({ locateFile: (file) => '/node_modules/rive-canvas/'+file, });
+  const loadFile = fetch(req).then((res) => res.arrayBuffer()).then(buf => new Uint8Array(buf));
```

Comment:
Since this is an `async` function, why not make this fully async by using `async/await` here, and return a `File` instead of a `Promise`?

Reply:
`loadRive` & `loadFile` can be performed in concurrency as they don't depend on each other. 
The `await` should be done on the `Promise.all` to be the most efficient in timing.